lib/src/file_picker_io.dart

```diff
@@ -73,7 +72,7 @@ class FilePickerIO extends FilePicker {
     bool? withData,
     bool? withReadStream,
   ) async {
-    final String type = describeEnum(fileType);
+    final String type = fileType.name;
```

Comment:
`decribeEnum()` has been deprecated for a while, in favor of the `name` property on enums themselves.